### PR TITLE
fix of undefined name of project

### PR DESCRIPTION
**Pull request by pavertomato**
seems like some of internal google's project may contain incorrect characters  and won't be parsed in json response

### CODE REVIEW

File: platform/viewer/src/googleCloud/ProjectsList.js [hunk 1]
```diff
@@ -65,7 +65,11 @@ class ProjectsList extends Component {
     }
     const body = (
       <tbody id="ProjectList">
-        {projects.filter(project => (project.name.toLowerCase().includes(filter.toLowerCase()) || filter=="")).map(this.renderTableRow)}
+        {
+          projects.filter(project => ( typeof project.name  === 'string' &&
+            (filter=="" || project.name.toLowerCase().includes(filter.toLowerCase()))
+          )).map(this.renderTableRow)
+        }
```
Comment (dannyrb) [inline platform/viewer/src/googleCloud/ProjectsList.js]: Can you break this down outside of the render function so it's a bit easier to reason about?
Something like:

```js
const lowerCaseFilter = filter.toLowerCase();
const filteredProjects = projects.filter(project => 
    typeof project.name  === 'string' &&
    (filter === "" || project.name.toLowerCase().includes(lowerCaseFilter));

...

{ filteredProjects.map(this.renderTableRow) }
```

Comment (pavertomato) [reply platform/viewer/src/googleCloud/ProjectsList.js]: not sure if got this right, i've moved filtering from body generation

Comment (dannyrb) [reply platform/viewer/src/googleCloud/ProjectsList.js]: If you've tested this, then I'm happy to merge. Comment and let me know, and I'll pull the merge lever.

Comment (pavertomato) [reply platform/viewer/src/googleCloud/ProjectsList.js]: sure, i've tested it, also have tested in docker image
also i may deploy on publicly available instance

Comment (dannyrb) [reply platform/viewer/src/googleCloud/ProjectsList.js]: If the PR is coming from a branch off of `OHIF/Viewers`, you can approve this workflow and a test docker image will be published:

image

**Pull request merged**